JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt

```diff
@@ -48,39 +47,31 @@ import com.example.jetnews.ui.lightThemeColors
 @Composable
 fun PostCardTop(post: Post) {
 // TUTORIAL CONTENT STARTS HERE
-    val typography = MaterialTheme.typography()
-    Column(modifier = LayoutWidth.Fill + LayoutPadding(16.dp)) {
+    val typography = MaterialTheme.typography
+    Column(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
         post.image?.let { image ->
-            // FIXME (dev07): This duplicate sizing is a workaround for the way Draw modifiers work in
-            // dev06. In a future release (dev07?) replace this with a single Image composable with a
-            // Clip modifier applied.
-
-            // This sizing is currently applied twice to make the image expand propertly inside the
-            // Clip.
-            val sizeModifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill
-            Container(modifier = sizeModifier) {
-                Clip(shape = RoundedCornerShape(4.dp)) {
-                    val imageModifier = ImagePainter(image).toModifier(scaleFit = ScaleFit.FillHeight)
-                    Box(modifier = sizeModifier + imageModifier)
-                }
-            }
+            val imageModifier = Modifier
+                .preferredHeightIn(minHeight = 180.dp)
+                .fillMaxWidth()
+                .clip(shape = RoundedCornerShape(4.dp))
+            Image(image, modifier = imageModifier, scaleFit = ScaleFit.FillHeight)
         }
-        Spacer(LayoutHeight(16.dp))
+        Spacer(Modifier.preferredHeight(16.dp))
 
-        val emphasisLevels = EmphasisLevels()
-        ProvideEmphasis(emphasis = emphasisLevels.high) {
+        val emphasisLevels = EmphasisAmbient.current
+        ProvideEmphasis(emphasisLevels.high) {
             Text(
                 text = post.title,
                 style = typography.h6
             )
         }
-        ProvideEmphasis(emphasis = emphasisLevels.high) {
+        ProvideEmphasis(emphasisLevels.high) {
```

Comment:
As both `Text`s are the same emphasis, you can use a single `ProvideEmphasis` with 2 children.

Reply:
Done!